src/main/java/edu/ucsb/cs156/happiercows/jobs/RecordCommonStatsJob.java

```diff
@@ -0,0 +1,39 @@
+package edu.ucsb.cs156.happiercows.jobs;
+
+import edu.ucsb.cs156.happiercows.entities.Commons;
+import edu.ucsb.cs156.happiercows.entities.CommonStats;
+import edu.ucsb.cs156.happiercows.repositories.CommonsRepository;
+import edu.ucsb.cs156.happiercows.services.CommonStatsService;
+import edu.ucsb.cs156.happiercows.services.jobs.JobContext;
+import edu.ucsb.cs156.happiercows.services.jobs.JobContextConsumer;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+
+//compute the stats for all games in progress and create one new row in the CommonsStats table for each commons. 
+//It will use the Average Cow Health Service to compute the cowhealth for each commons.
```

Comment:
If we are going to have a comment here, let's format it as a Javadoc comment:

```suggestion
/** This job computes the stats for all games in progress and creates one new row in the CommonsStats table for each commons.   It uses the Average Cow Health Service to compute the cowhealth for each commons.
*/
```